src/components/app/ProfileViewerContainer.css

```diff
@@ -0,0 +1,17 @@
+.ProfileViewerContainer .layout-pane > * {
```

Comment:
Here I consider it's not expensive because the part `.layout-pane > *` at the end fails fast (selectors are generally applied right to left, unless there are more shortcuts). But I may be wrong, and I'd be glad to have @mstange's advice too :)